R/RWR_LOE.R

```diff
@@ -157,11 +157,17 @@ view_top_network_loe <- function(results,
                                  cyto = 1,
                                  modname = "") {
   if (cyto) {
-    topresults <- RandomWalkRestartMH::create.multiplexNetwork.topResults(
-      results,
-      nw_mpo,
-      k = ntop
-    )
+    #1.  compress multiplex to aggregate net
+    merged_mp <- merged_with_edgecounts(nw_mpo)$merged_network
```

Comment:
Per conversation with @dkainer - inverse rank product might be a worthwhile merging method to move forward with: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0106319